script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -93,6 +93,7 @@ Keeping up to date with what happened at Souya station isn't just Arihiko amusin
 [sha:05db94f2340283565594bb9c30d13f70ef95bb81]{
 -- Page 96, Offset 16809.
 -- 店員、刃物で腹部を刺され重傷、犯人はアルコール依存症。うへぇ痛そー。酔っ払いに<話|はなし>通じねえもんなぁ。やっぱ買い物は通販にかぎりますなあ」
+// to be rechecked, set phrase
```

Comment:
Something about にかぎる I think?

Reply:
Right, Petrikow said that かぎります in this case is "the best", not "limited to"
> やっぱ買い物は通販にかぎりますなあ
I guess shopping online really is your [superlative] [option/choice/bet, depending on superlative choice].

Maybe "safest" given the context? Otherwise "best" is a safe fallback